documentation/guides/introduction/01_quick_start.md

```diff
@@ -0,0 +1,169 @@
+- [Aleo123](https://testnet.aleo123.io/)
+- [Aleo.Info](https://testnet.aleo.info/)
+<!-- markdown-link-check-enable -->
+
+:::info
+Depending on the size of your program, you may not have sufficient credits to deploy your program if you are using an account funded with only one faucet transaction.  If that's the case, you can still interact with a program already deployed to the Testnet.
+:::
+
+## 5. Execute a transaction
+
+You can now use your deployed token program!  
+
+:::tip
+If you did not have enough funds to deploy a program, you can use the `token_quickstart.aleo` program to test the execution of the token program methods.
+:::
+
+When you call a method from your program, the logic is executed locally and is accompanied with a zero-knowledge proof that attests to the correctness of the program execution and its corresponding outputs.  This proof does not reveal any information about the input values.  If the proof is verified by the validators, then the blockchain state is updated.  
+
+* Minting a private token
+
+Minting a private token entails creating a Record.  Records are data structures that are stored on-chain as ciphertexts and can only be decrypted using the View Key, which is derived from the Record owner's Private Key.  This ensures that only the owner of a Record has the ability to decrypt it.  
+
+To mint a private token, navigate to the Execute widget in the Leo Playground and enter the name of your program in the Program ID tab.  Alternatively, you can use the deployed token program 'token_quickstart.aleo`.  Click the magnifying glass icon to confirm that the Program ID corresponds to a deployed program.
+
+![execute](images/deploy.png)
+
+Next, enter your Private Key and select the `mint_private` method from the Function drop-down menu.  In the Inputs tab, enter your Account Address and a `u64` integer.
+
+```
+["ALEO_ADDRESS_HERE", "50u64"]
+```
+Click Estimate Fee followed by Execute.  As before with the deploy transaction, you can copy and past your transaction ID in a block explorer.  You should see something resembling the following:
+
+![mint_private](images/mint_private.png)
+
+:::info
+The output of the transaction is a Record.  In order to decrypt the Record, you need to copy and past the ciphertext, navigate to the Records widget in the Leo Playground, enter the Record ciphertext along with the View Key that corresponds to your Account's Private Key, and click Decrypt.  The decrypted Record should resemble the following:
+```{
+  owner: aleo1kypwp5m7qtk9mwazgcpg0tq8aal23mnrvwfvug65qgcg9xvsrqgspyjm6n.private,
+  amount: 50u64.private,
+  _nonce: 3175255370513411091535466147458245312227668453916963245036391157478647265587group.public
+}
+```
+To learn more about Records, click [here](../../concepts/fundamentals/02_records.md)
+:::
+
+* Mint a public token
+
+Public tokens are stored in mappings.  Unlike Records, which are encrypted on-chain, mappings will always contain public data.  
```

Comment:
```suggestion
In the Token program, public balances are stored in program mappings. Mappings are public key-value stores on Aleo Network nodes which store long term persistent public state onchain.

Below we will mint a public token using the `mint_public` function. When this function is run, it will update the balance in the `account` mapping corresponding to the specified Aleo `address`. 
```